### PR TITLE
chore: try fix CI coverage

### PR DESCRIPTION
**Pull request by Ted-Jiang**
# Which issue does this PR close?

test for fix ci.

Closes #.

 # Rationale for this change

# What changes are included in this PR?

# Are there any user-facing changes?

# Does this PR break compatibility with Ballista?

### CODE REVIEW

File: datafusion/core/tests/sql/explain_analyze.rs [hunk 1]
```diff
@@ -737,6 +737,7 @@ async fn test_physical_plan_display_indent_multi_children() {
 }
 
 #[tokio::test]
+#[cfg_attr(tarpaulin, ignore)]
```
Comment (Ted-Jiang) [inline datafusion/core/tests/sql/explain_analyze.rs:740]: These 4 tests run fail in `coverage`(don't know why 🤕 ), so i try to ignore them in coverage, but they still check in `cargo test`.

Comment (alamb) [reply datafusion/core/tests/sql/explain_analyze.rs:740]: How did they fail?

**Review by alamb — COMMENTED** [review]
LGTM -- thanks @Ted-Jiang

**Review by alamb — APPROVED** [review]

**Review by tustvold — APPROVED** [review]
This seems like a pragmatic solution to the issue

**Pull request merged**